zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -239,7 +239,7 @@ func (ob *Observer) ProcessInboundTrackers(ctx context.Context) error {
 			return err
 		}
 		ob.logger.Inbound.Info().
-			Msgf("Vote submitted for inbound Tracker, Chain : %s,Ballot Identifier : %s, coin-type %s", ob.Chain().ChainName, ballotIdentifier, coin.CoinType_Gas.String())
+			Msgf("Vote submitted for inbound Tracker, Chain : %s,Ballot Identifier : %s, coin-type %s", ob.Chain().Name, ballotIdentifier, coin.CoinType_Gas.String())
```

Comment:
Let's move all vars to log fields e.g. `Str("inbound.chain", chain.Name)`

Reply:
I can do it here but it will require another PR for a global change

Reply:
7ddc7f26d